.circleci/build_config.yml

```diff
@@ -544,6 +538,8 @@ step-electron-build: &step-electron-build
         (cd out/Default; zip mksnapshot.zip mksnapshot_args clang_x64_v8_arm64/gen/v8/embedded.S)
         rm -rf out/Default/clang_x64_v8_arm64/gen
         rm -rf out/Default/clang_x64_v8_arm64/obj
+        rm -rf out/Default/clang_x64_v8_arm64/thinlto-cache
+        rm -rf out/Default/clang_x64_v8_arm64/v8_context_snapshot_generator.dSYM
```

Comment:
We still need this

Reply:
@MarshallOfSound Just the .dSYM file, right (the LTO cache is still fine to blow away)? I'll remove this, thanks for catching 👍